plugin.py

```diff
@@ -17,7 +17,8 @@ def plugin_info(self):
             maturity=MaturityLevel.PROOF_OF_CONCEPT,
             webpage_url='',  # e.g. url to the code repository of your plugin
             matcher='$data.type=raw',  # add the query for the types of traces your plugin should match
-            license='Apache License 2.0'
+            license='Apache License 2.0',
+            resources=PluginResources(maximum_cpu=2, maximum_memory=512, maximum_workers=6)
```

Comment:
Maybe add a trailing comma to avoid triggering diffs for any additions later down the line (like the license here)?

```suggestion
            resources=PluginResources(maximum_cpu=2, maximum_memory=512, maximum_workers=6),
```